crates/apub_objects/src/protocol/tags.rs

```diff
@@ -0,0 +1,43 @@
+use lemmy_db_schema::{
+  newtypes::CommunityId,
+  source::tag::{Tag, TagInsertForm},
+};
+use serde::{Deserialize, Serialize};
+use url::Url;
+
+/// The [ActivityStreams vocabulary](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-tag)
+/// defines that any object can have a list of tags associated with it.
+/// Tags in AS can be of any type, so we define our own types.
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Default)]
+enum CommunityTagType {
+  #[default]
+  PostTag,
```

Comment:
Couldn't tags owned at tho community level potentially be applied to other things in the future (comments)? In that sense it just makes sense to call them community tags and let them be applied to anything owned by the community.